netlify.toml

```diff
@@ -21,147 +21,182 @@ publish = "site"
-      style-src 'self' 'unsafe-inline' https://fonts.googleapis.com/; \
-      worker-src 'self'
+# Content-Security-Policy to prevent XSS attacks.
+#
+# default-src
+#   'none' - all resources from current origin are blocked by default
+# connect-src
+#   'self' - all connections to current current origin are permitted
+#   https://api.github.com - required to display release information
+#   https://api.segment.io - default domain for Segment analytics
+#   https://cdn.segment.com - default domain for Segment analytics.js
+#   https://api.segment.prefect.io - custom proxy for Segment analytics
+#   https://cdn.segment.prefect.io - custom proxy for Segment analytics.js
+#   https://www.google-analytics.com - Google Analytics
+# font-src
+#   https://fonts.gstatic.com - load fonts from Google Fonts
+# frame-ancestors
+#   'none' - prevent embedding as a frame
+# frame-src
+#   'none' - prevent framing by default
+#   https://www.youtube.com - allow embedding of YouTube videos
+# img-src
+#   'self' - all images from current origin are permitted
+#   data: - images embedded inline are permitted
+#   https://avatars.githubusercontent.com - embed user images from GitHub
+# media-src
+#   'none' - no audio or video files are permitted
+# object-src
+#   'none' - no legacy objects are allowed; see
+#            https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/object-src
+# script-src
+#   'self' - all resources from current origin are permitted
+#   'unsafe-inline' - required for scripts loaded in partials
+#   https://cdn.segment.com - default domain for Segment analytics.js
+#   https://cdn.segment.prefect.io - custom proxy for Segment analytics.js
+#   https://www.googletagmanager.com - allow snippet customization using Google Tag Manager
+# style-src
+#   'self' - all styles from current origin are permitted
+#   'unsafe-inline' - required for styles loaded in partials
+# worker-src
+#   'self' - allow web workers from current origin
+Content-Security-Policy = """\
+      default-src \
+        'none' \
+        ; \
+      connect-src \
+        'self' \
+        https://api.github.com \
+        https://api.segment.io \
+        https://cdn.segment.com \
+        https://api.segment.prefect.io \
+        https://cdn.segment.prefect.io \
+        https://www.google-analytics.com \
+        ; \
+      font-src 
+        https://fonts.gstatic.com \
+        ; \
+      frame-ancestors 'none' \
+        ; \
+      frame-src 
+        https://www.youtube.com \
+        ; \
+      img-src 'self' \
+        data: \
+        https://avatars.githubusercontent.com \
+        ; \
+      media-src \
+      'none' \
+        ; \
+      object-src \
+        'none' \
+        ; \
+      script-src \
+        'self' \
+        'unsafe-inline' \
+        https://cdn.segment.com/ \
+        https://cdn.segment.prefect.io \
+        https://www.googletagmanager.com \
+        https://unpkg.com/mermaid \
```

Comment:
This is the only actual new content - everything else is a difference in TOML formatter between me and @jawnsy